src/controllers/update/confirm.overseas.entity.details.controller.ts

```diff
@@ -46,13 +52,19 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
     const isRemove: boolean = await isRemoveJourney(req);
+    const appData: ApplicationData = await getApplicationData(req.session as Session);
 
     if (isRemove) {
       return res.redirect(`${config.OVERSEAS_ENTITY_PRESENTER_URL}${config.JOURNEY_REMOVE_QUERY_PARAM}`);
     }
 
-    if (isActiveFeature(config.FEATURE_FLAG_ENABLE_RELEVANT_PERIOD)) {
-      return res.redirect(config.RELEVANT_PERIOD_OWNED_LAND_FILTER_URL + config.RELEVANT_PERIOD_QUERY_PARAM);
+    if (appData.entity && appData.entity_number) {
+      const statements: CompanyPersonsWithSignificantControlStatements = await getCompanyPscStatements(req, appData.entity_number);
+      relevantPeriodStatementsState.has_answered_relevant_period_question = !!(statements && statements?.items?.length > 0);
```

Comment:
Could this be simplified to the below?
```suggestion
      relevantPeriodStatementsState.has_answered_relevant_period_question = statements?.items?.length > 0;
```